domain/allocine.py

```diff
@@ -3,19 +3,24 @@
 from connectors.api_allocine import get_movies_showtimes_from_allocine, get_movie_poster_from_allocine
 from utils.logger import logger
 
+MOVIE_TYPE = 'SPECIAL_EVENT'
```

Comment:
Je la renommerais bien en MOVIE_SPECIAL_TYPE ou SPECIAL_TYPE

Reply:
Pourquoi pas ? Le vocabulaire d'allociné, c'est special event, j'éviterais de le transformer :)
```suggestion
MOVIE_SPECIAL_EVENT = 'SPECIAL_EVENT'
```

Reply:
👍